distribution/src/assembly/distribution.xml

```diff
@@ -34,6 +34,14 @@
             <outputDirectory>${file.separator}</outputDirectory>
             <unpack>true</unpack>
         </dependencySet>
+        <dependencySet>
+            <includes>
+                <include>*:iotdb-grafana:*</include>
+            </includes>
+            <outputDirectory>tools/grafana/</outputDirectory>
+            <outputFileNameMapping>${artifact.artifactId}.${artifact.extension}</outputFileNameMapping>
+            <unpack>false</unpack>
```

Comment:
should we also include the config here? This is something I forgot in 0.9.3 branch. WDYT?

Reply:
needed and have corrected.